pkg/controller/registry/resolver/resolver_test.go

```diff
@@ -1361,7 +1361,7 @@ func TestSolveOperators_WithSkips(t *testing.T) {
 	namespace := "olm"
```

Comment:
Should this be a separate test case, so that we have two? Skips without startingCSV, skips with startingCSV?

Reply:
The non-startingCSV is implied with the dependent operator but I made 2 test cases now just to be more clear.